card/mod/05_standard/spec/set/all/history_spec.rb

```diff
@@ -164,7 +168,9 @@
     context 'for plus card' do
       before do
         Card::Auth.as_bot do
+          content = "Nobody expects the Spanish inquisition"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.